dev_run.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+#sudo gameserver/docker4bots/00_setup_shm_for_test.sh
```

Comment:
`sudo` is not needed here, as `00_setup_shm_for_test.sh` uses it where necessary. That script might even produce wrong permissions if it runs as root, so please remove the `sudo` here and in `README.md`.

Reply:
sorry, I didn't see the PR ( https://github.com/schlangenprogrammiernacht/gameserver/pull/2 ) in the gameserver repo. Let's discuss this issue there.